tools/shared.py

```diff
@@ -2208,7 +2208,7 @@ def parse_symbols(output, include_internal=False):
         continue # e.g.  filename.o:  , saying which file it's from
       parts = [seg for seg in line.split(' ') if len(seg)]
       # pnacl-nm will print zero offsets for bitcode, and newer llvm-nm will print present symbols as  -------- T name
-      if len(parts) == 3 and parts[0] == "--------" or re.match(r'^\d\d\d\d\d\d\d\d$', parts[0]):
+      if len(parts) == 3 and parts[0] == "--------" or re.match(r'^[\dabcdef][\dabcdef][\dabcdef][\dabcdef][\dabcdef][\dabcdef][\dabcdef][\dabcdef]$', parts[0]):
```

Comment:
Why this?  

Can you do `[\da-f]` instead?

Reply:
also: `[\da-f]{8}$`?

Reply:
thanks, yeah, that's nicer.